### ISSUE TITLE
Python 3.8.0 compile issue

### ISSUE THREAD

**Issue by NickCuso** [ethereum/vyper#1641]
### Version Information

* vyper Version (output of `vyper --version`): 0.1.0b13+commit.375fb27
* OS: linux
* Python Version (output of `python --version`): 3.8.0
* Environment (output of `pip freeze`): 
```
asn1crypto==0.24.0
asttokens==1.1.13
Brlapi==0.6.6
certifi==2018.1.18
chardet==3.0.4
command-not-found==0.3
cryptography==2.1.4
defer==1.0.6
httplib2==0.9.2
idna==2.6
keyring==10.6.0
keyrings.alt==3.0
language-selector==0.1
louis==3.5.0
netifaces==0.10.4
olefile==0.45.1
onboard==1.4.1
pexpect==4.2.1
Pillow==5.1.0
pycairo==1.16.2
pycrypto==2.6.1
pycryptodome==3.8.2
pygobject==3.26.1
python-apt===1.6.4-elementary6-ubuntu5.0.1
pyxdg==0.25
PyYAML==3.12
QtPy==1.3.1
reportlab==3.4.0
requests==2.18.4
requests-unixsocket==0.1.5
SecretStorage==2.3.1
six==1.11.0
ubuntu-drivers-common==0.0.0
ufw==0.36
urllib3==1.22
vyper==0.1.0b13
xkit==0.0.0
```

### What's your issue about?

Compile fails on `constant`.

Please include information like:

* full output of the error you received
```
Error: vyper.exceptions.SyntaxException: line 6:30 Invalid syntax (unsupported "Constant" Python AST node).
     5
---> 6 MAX_UINT: constant(uint256) = 2**256 - 1
-------------------------------------^
     7 # @notice The max possible value

Compilation of /home/circleci/repo/contracts/BigMath.vy failed. See above.
    at Object.compile (/home/circleci/repo/node_modules/truffle/build/webpack:/packages/workflow-compile/legacy/index.js:72:1)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Truffle v5.0.40 (core: 5.0.40)
Node v10.16.3
```
* what command you ran
`truffle compile`

* the code that caused the failure (see [this link](https://help.github.com/articles/basic-writing-and-formatting-syntax/) for help with formatting code)
`MAX_UINT: constant(uint256) = 2**256 - 1`

### How can it be fixed?

Fill this in if you know how to fix it.

Workaround by using an older version of Python.  Tested with 2.7.15 and 3.7.4
Ideally the latest version, 3.8.0, would work as well.

**Comment by iamdefinitelyahuman** [comment]
This is a result of Python 3.8 replacing various node classes with `ast.Constant` ([related docs](https://docs.python.org/3/library/ast.html#node-classes)).  I have a quick fix at [iamdefinitelyahuman/vyper:py38-ast](https://github.com/iamdefinitelyahuman/vyper/tree/py38-ast), but it cannot be properly tested until https://github.com/pytoolz/cytoolz/issues/136 merges.

**Comment by charles-cooper** [comment]
@iamdefinitelyahuman is this resolved?

**Comment by iamdefinitelyahuman** [comment]
Yes, it should be resolved in `0.1.0b14`, we can close this
